web/src/components/BlockVisualizer.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-list-item
+    class="no-y-padding block-visualizer"
+    dense
+    :disabled="!block.active"
+    :style="dummy ? 'visibility: hidden' : ''"
```

Comment:
What is the dummy used for?

Reply:
`dummy` is used when the component is set to invisible but still needs to take up the same amount of space.